pkgs/development/libraries/audio/libgme/default.nix

```diff
@@ -21,4 +21,12 @@ in stdenv.mkDerivation {
   };
 
   buildInputs = [ cmake ];
+
+  # It used to reference it, in the past, but thanks to the postFixup hook, now
+  # it doesn't.
+  disallowedReferences = [ stdenv.cc.cc ];
+
+  postFixup = stdenv.lib.optionalString stdenv.isLinux ''
+    ${removeReferencesTo}/bin/remove-references-to -t ${stdenv.cc.cc} "$(readlink -f $out/lib/libgme.so)"
```

Comment:
Not sure this will work correctly when cross-compiling, as it won't automatically use the build-platform `removeReferencesTo` — instead, it should be placed in `nativeBuildInputs` and used via PATH lookup.

Reply:
:+1:.